packages/base-styles/_variables.scss

```diff
@@ -104,3 +104,16 @@ $block-selected-to-content: $block-edge-to-content - $block-selected-margin - $b
 $radius-round-rectangle: 4px;
 $radius-round: 50%;
 $radius-block-ui: 2px;
+
+:root {
+	--wp-theme-color: theme(primary);
+	--wp-theme-color-secondary: theme(secondary);
+	--wp-theme-color-toggle: theme(toggle);
+
+	// Ideally this are computed and not their own variables but due to Current CSS limitation
+	// We need these.
+	--wp-theme-color-darker-20: theme(primaryDarker20);
+	--wp-theme-color-darker-10: theme(primaryDarker10);
+	--wp-theme-color-lighter-10: theme(primaryLighter10);
+	--wp-theme-color-lighter-40: theme(primaryLighter40);
```

Comment:
the problem here is that there's no way to lighten/darked colors in CSS (without SASS or PostCSS).

I'd like to remove these variations as soon as possible but it requires changes to the styles theme selves, so I added these, for now, to avoid breaking the existing styles. Any help to refactor the styles relying on these will be appreciated.